core/src/main/java/org/apache/brooklyn/core/mgmt/rebind/PeriodicDeltaChangeListener.java

```diff
@@ -382,6 +401,8 @@ protected void persistNowInternal(boolean alreadyHasMutex) {
             if (!alreadyHasMutex) persistingMutex.acquire();
             if (!isActive() && state != ListenerState.STOPPING) return;
             
+            updatePlaneIdIfTimedOut();
```

Comment:
I agree it makes sense but would do it for all writes, not just the planeId. Not comfortable with doing it just for planeId - would break the existing assumption about writes.